integration-cli/docker_cli_rename_test.go

```diff
@@ -9,28 +9,14 @@ import (
 )
 
 func (s *DockerSuite) TestRenameStoppedContainer(c *check.C) {
-	runCmd := exec.Command(dockerBinary, "run", "--name", "first_name", "-d", "busybox", "sh")
-	out, _, err := runCommandWithOutput(runCmd)
-	if err != nil {
-		c.Fatalf(out, err)
-	}
+	out, _ := dockerCmd("run", "--name", "first_name", "-d", "busybox", "sh")
```

Comment:
@HuKeping You missed the ``c`` argument here (and in the others): should be like : ``dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", "sh")``